lib/shared/permissions.ts

```diff
@@ -40,7 +40,7 @@ function sudoExecAsync(
 			cmd,
 			options,
 			(error: Error | undefined, stdout: Std, stderr: Std) => {
-				if (error !== undefined) {
+				if (error) {
```

Comment:
While this is a functional workaround for the issue, this code wasn't actually wrong. The real bug is here:
https://github.com/jorangreef/sudo-prompt/blob/v9.2.1/index.js#L159-L191

It passes the error argument of the `child_process.exec()` callback, which is [documented](https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback) to be `null` on success, directly to the callback of `sudoPrompt.exec()`, which violates [its typescript declaration](https://github.com/jorangreef/sudo-prompt/blob/v9.2.1/index.d.ts#L4).